G1ANT.Addon.Selenium/Api/SeleniumManager.cs

```diff
@@ -216,14 +216,13 @@ private static IWebDriver CreateNewWebDriver(string webBrowserName, BrowserType
                     break;
 
                 case BrowserType.Edge:
-                    var edgeService = Edge.EdgeDriverService.CreateDefaultService(driversDirectory);
+                    var edgeService = Edge.EdgeDriverService.CreateDefaultService();
```

Comment:
It would be great if you refactor this method and move all code from each case to separate method